src/op_mode/image_installer.py

```diff
@@ -521,6 +521,10 @@ def get_cli_kernel_options(config_file: str) -> list:
     if 'quiet' in kernel_options:
         cmdline_options.append('quiet')
 
+    # Early reboot on kernel panic via kernel cmdline (must match system_option.py)
+    if dict_search('system.option.reboot-on-panic', config_dict) is not None:
```

Comment:
In order to keep the current coding style for this feature please change this to: 

```suggestion
   if 'reboot-on-panic' in kernel_options:
```

Or is there any specific reason to use `dict_search()`? I see you used it in good intention - but here all code paths use a different style which we should  keep for the moment.